README.md

```diff
@@ -26,92 +52,232 @@ def workflow()
+
+####
+
+Pause your workflow executions until a notification is received, or emit events from your workflow to send progress updates to external clients.
+All notifications are stored in Postgres, so they can be send and received with exactly-once semantics.
+Set durable timeouts when waiting for events, so you can wait for as long as you like (even days or weeks) through interruptions or restarts, then resume once the notification arrives or the timeout is reached.
+
+For example, build a reliable billing workflow that durably waits for a notification from a payments service, processing it exactly-once:
+
+```python
+@DBOS.workflow()
+def billing_workflow():
+  ... # Calculate the charge, then submit the bill to a payments service
+  payment_status = DBOS.recv(PAYMENT_STATUS, timeout=payment_service_timeout)
+  if payment_status is not None and payment_status == "paid":
+      ... # Handle a successful payment.
+  else:
+      ... # Handle a failed payment or timeout.
 ```
 
-Save the program into `main.py` and start it with `fastapi run`.
-Visit `localhost:8000` in your browser to start the workflow.
-When prompted, press `Control + C` (You may need to press `Control + C` twice quickly, or press `Control + \`, if `Control + C` is not effective in your environment) to force quit your application.
-It should crash midway through the workflow, having completed step one but not step two.
-Then, restart your app with `fastapi run`.
-It should resume the workflow from where it left off, completing step two without re-executing step one.
+</details>
+
+
+## Getting Started
 
-To learn how to build more complex workflows, see the [programming guide](https://docs.dbos.dev/python/programming-guide) or [examples](https://docs.dbos.dev/examples).
+To get started, follow the [quickstart](https://docs.dbos.dev/quickstart) to install the open-source library and connect it to a Postgres database.
+Then, check out the [programming guide](https://docs.dbos.dev/python/programming-guide) to learn how to build with durable workflows and queues.
 
 ## Documentation
 
 [https://docs.dbos.dev](https://docs.dbos.dev)
 
 ## Examples
 
+[https://docs.dbos.dev/examples](https://docs.dbos.dev/examples)
+
+## DBOS vs. Other Systems
+
+<details><summary><strong>DBOS vs. Temporal</strong></summary>
+
+####
+
+Both DBOS and Temporal provide durable execution, but DBOS is implemented in a lightweight Postgres-backed library whereas Temporal, and architecturally similar systems such as Restate and Inngest, are implemented in an externally orchestrated server.
+
+You can add DBOS to your program by installing the open-source library, connecting it to Postgres, and annotating workflows and steps.
+By contrast, to add Temporal to your program, you must rearchitect your program to move your durable workflows to a Temporal worker, configure a Temporal server to orchestrate those workflows, and access your workflows only through a Temporal client.
```

Comment:
```suggestion
By contrast, to add Temporal to your program, you must rearchitect your program to move your workflows to a Temporal worker, configure a Temporal server to orchestrate those workflows, and access your workflows only through a Temporal client.
```

they are not durable yet :)

Reply:
Also: what about the performance argument, specifically for latency sensitive workflows where the RTT-in-the-wire slows down every step? (DBOS RTTs to Postgres, a Temporal app RTTs to the temporal server which itself RTTs to its persistence storage.